### ISSUE TITLE
Fetching statements through statements api causes a json operator error

### ISSUE THREAD

**Issue by BugVito** [adlnet/ADL_LRS#359]
To reproduce:
- From a fresh install
- Register a user
- POST a single statement using basic auth (simple or complex does not seem to make any difference)
- GET statements, using basic auth, at /xAPI/statements/

Note that pulling a simple statement by ID works perfectly fine.

Ubuntu 15.04
PostgreSQL 9.4.5
Python packages:
amqp==1.4.6
anyjson==0.3.3
bencode==1.0
billiard==3.3.0.21
celery==3.1.18
Django==1.4
django-endless-pagination==2.0
django-extensions==1.1.1
django-jsonify==0.3.0
httplib2==0.9.2
isodate==0.4.9
jsonfield==0.9.19
kombu==3.0.29
meld3==1.0.2
oauth2==1.5.211
psycopg2==2.5
pycrypto==2.5
pytz==2012rc0
rfc3987==1.3.4
shortuuid==0.4.2
six==1.10.0
South==1.0.1
supervisor==3.0a12
wheel==0.24.0

```
2015-10-29 14:02:06,503 [INFO] lrs.views: 
Exception while processing: /xAPI/statements/
2015-10-29 14:02:06,503 [ERROR] lrs.views: could not identify an equality operator for type json
LINE 1: ...t_score_min", "lrs_statement"."result_score_max", "lrs_state...
                                                             ^

Traceback (most recent call last):
  File "/home/adllrs/ADL_LRS/lrs/views.py", line 619, in handle_request
    return processors[path][req_dict['method']](req_dict)
  File "/home/adllrs/ADL_LRS/lrs/util/req_process.py", line 192, in statements_get
    resp, content_length = process_complex_get(req_dict)
  File "/home/adllrs/ADL_LRS/lrs/util/req_process.py", line 96, in process_complex_get
    stmt_result = complex_get(param_dict, limit, language, format, attachments)
  File "/home/adllrs/ADL_LRS/lrs/util/retrieve_statement.py", line 125, in complex_get
    return create_stmt_result(stmtset, stored_param, language, format)
  File "/home/adllrs/ADL_LRS/lrs/util/retrieve_statement.py", line 143, in create_stmt_result
    stmt_result = '{"statements": [%s], "more": ""}' % ",".join([json.dumps(stmt.full_statement) for stmt in stmt_set.order_by(stored)])
  File "/home/adllrs/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 118, in _result_iter
    self._fill_cache()
  File "/home/adllrs/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 875, in _fill_cache
    self._result_cache.append(self._iter.next())
  File "/home/adllrs/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 291, in iterator
    for row in compiler.results_iter():
  File "/home/adllrs/env/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 763, in results_iter
    for rows in self.execute_sql(MULTI):
  File "/home/adllrs/env/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 818, in execute_sql
    cursor.execute(sql, params)
  File "/home/adllrs/env/local/lib/python2.7/site-packages/django/db/backends/util.py", line 40, in execute
    return self.cursor.execute(sql, params)
  File "/home/adllrs/env/local/lib/python2.7/site-packages/django/db/backends/postgresql_psycopg2/base.py", line 52, in execute
    return self.cursor.execute(query, args)
DatabaseError: could not identify an equality operator for type json
LINE 1: ...t_score_min", "lrs_statement"."result_score_max", "lrs_state...
```

**Comment by ljwolford** [comment]
That is a known issue happening because you are using PostgreSQL 9.4.5 and the install instructions were written against 9.1. We do plan on updating both Django and Postgres in the future but not quite yet. Thank you for pointing it out and feel free to submit a fix if you'd like. For more information visit here: http://stackoverflow.com/questions/23509740/distinct-on-postgresql-json-data-column

**Comment by BugVito** [comment]
I did realize that the versions could be an issue when clicking the "send" button.
I had a very old version of your LRS running on ubuntu 15.04 and postgresql 9.4, and made the clumsy assumption that it should all be fine.  Sorry for this, and I actually may look into a fix when time permits.

As for a quick resolution, I downgraded to ubuntu 12.04, and it's version of postgresql, which is 9.1.

Thanks for a quick response!
